.circleci/config.yml

```diff
@@ -134,6 +144,24 @@ commands:
             - run:
                 command: cat ~/attached_workspace/environment_variables >> ${BASH_ENV}
                 name: Restore environment variables
+    restore_pods:
+        description: Restores and saves the cocoa pods cache.
+        steps:
+            - restore_cache:
+                keys:
+                    - 6-pods-{{ arch }}-{{ checksum "frontend/ios/Podfile.lock" }}
```

Comment:
do we really have such a high cache number for the pods? Since this is new shouldn't we start with `1-`